designer/server/src/routes/forms/create.js

```diff
@@ -310,20 +310,22 @@ export function redirectToStepWithErrors(request, h, error) {
  * @param {ResponseToolkit} h
  * @param {Error} [error]
  * @param {boolean} [redirectToPreviousStep] Optionally redirect to errors on previous steps, else it uses the current URL
+ * @param {string} [flashKey] Optionally redirect to errors on previous steps, else it uses the current URL
```

Comment:
Nice one @KiranVarma02

Just need to tell the compile that only known session keys are allowed

```suggestion
 * @param {keyof typeof sessionNames} [flashKey] Optionally redirect to errors on previous steps, else it uses the current URL
```

Reply:
done